src/client/components/household-information/FinancialDisclosureSection.jsx

```diff
@@ -73,8 +74,8 @@ class FinancialDisclosureSection extends React.Component {
                 errorMessage={isValidFinancialDisclosure(this.props.data) ? '' : 'Please acknowledge this requirement'}
                 label="I understand VA is not currently enrolling new applicants who decline to provide their financial information unless they have other qualifying eligibility factors."
                 name="understandsFinancialDisclosure"
-                checked={this.props.data.understandsFinancialDisclosure}
-                onValueChange={(update) => {this.props.onStateChange('understandsFinancialDisclosure', update);}}/>
+                checked={this.props.data.understandsFinancialDisclosure.value}
+                onValueChange={(update) => {this.props.onStateChange('understandsFinancialDisclosure', { value: update, dirty: false });}}/>
```

Comment:
Do you need to pass `{ value: update, dirty: false }`? I think just `update` will work.